source-code/core/app/src/createInlang.ts

```diff
@@ -75,12 +75,20 @@ export const createInlang = async (args: {
 			if (!conf) return
 			loadModules({ config: conf, nodeishFs: args.nodeishFs, _import: args._import })
 				.then((resolvedModules) => {
+					if (resolvedModules.errors.length) {
+						throw new Error(resolvedModules.errors as any)
+					}
+					if (!resolvedModules.runtimePluginApi.loadMessages || !resolvedModules.runtimePluginApi.saveMessages) {
+						throw new Error("It seems you did not install any plugin that handles messages. Please add one to make inlang work.") // TODO: add link to docs
```

Comment:
@inlang/team do we agree to throw in these cases? If yes, I can add tests and a meaningful error description tomorrow.

Reply:
I think we should never throw in `createInlang` and instead implement a mock readMessages, saveMessages, and push the error to `inlang.errors`: 

- more explicit error handling without a "gotcha" 
- apps can implement "create a new project" workflows easier by reacting to the pushed `inlang.errors` instead of handling try catch things.

Reply:
I got confused by this line:
![image](https://github.com/inlang/inlang/assets/21335119/cb4f826a-ce8e-40d1-8a8e-65adb6d495fa)

Yes, so we add a new error to the array 👍

Reply:
fixed, pushed but not tested yet